services/app/src/components/views/Search.test.tsx

```diff
@@ -39,6 +41,22 @@ describe('Search', () => {
     return {props, e: render(<Search {...(props as any as Props)} />, undefined /*renderOptions*/)};
   }
 
+  function setupMount(overrides?: Partial<Props>) {
```

Comment:
This is a lot of duplicate code IMO... easy for it to drift (for example, onSearch only exists in `setupMount`).

I'd prefer if you changed the original `setup()` to `mount` the component, then where we need to render things just do `setup().render()` or equivalent.